test/functional/apps/dashboard/dashboard_saved_query.js

```diff
@@ -58,7 +58,7 @@ export default function ({ getService, getPageObjects }) {
         await savedQueryManagementComponent.openSavedQueryManagementComponent();
         const descriptionText = await testSubjects.getVisibleText('saved-query-management-popover');
         expect(descriptionText).to.eql(
-          'SAVED QUERIES\nThere are no saved queries. Save query text and filters that you want to use again.\nSave current query'
+          'Saved Queries\nThere are no saved queries. Save query text and filters that you want to use again.\nSave current query'
```

Comment:
Intentional change?

Reply:
This is the side effect of https://github.com/opensearch-project/oui/pull/1283